components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c

```diff
@@ -356,11 +372,11 @@ static lv_display_t *lvgl_port_add_disp_priv(const lvgl_port_display_cfg_t *disp
 
 err:
     if (ret != ESP_OK) {
-        if (buf1) {
-            free(buf1);
+        if (disp_ctx->draw_buffs[0]) {
+            free(disp_ctx->draw_buffs[0]);
         }
-        if (buf2) {
-            free(buf2);
+        if (disp_ctx->draw_buffs[1]) {
+            free(disp_ctx->draw_buffs[1]);
```

Comment:
Thank you for this fix! It was already fixed in the master branch, so you might have to resolve some conflicts...